flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceContinuous.java

```diff
@@ -75,7 +75,8 @@ public class TestIcebergSourceContinuous {
   @Test
   public void testTableScanThenIncremental() throws Exception {
     GenericAppenderHelper dataAppender =
-        new GenericAppenderHelper(tableResource.table(), FileFormat.PARQUET, TEMPORARY_FOLDER);
+        new GenericAppenderHelper(
+            tableResource.table(), FileFormat.PARQUET, TEMPORARY_FOLDER.getRoot().toPath());
```

Comment:
all of these changes should go away once `GenericAppenderHelper` is reverted

Reply:
these changes were reverted, pls see my response at the previous comment